### DISCUSSION TITLE
nextjs13 client side not getting data after refresh I am using params to get id and then fetch product details

### DISCUSSION THREAD

**Discussion by Lalman888** [vercel/next.js#53974] [App Router]
I am trying to fetch the data when I have id http://localhost:3000/product/64d630a5548f39ac24bb5d3e  when I go  http://localhost:3000/product to this  http://localhost:3000/product/64d630a5548f39ac24bb5d3e I get the data but when I directly go to this link  http://localhost:3000/product/64d630a5548f39ac24b  I did not get the data. In API  I am not able to get response.
<img width="579" alt="Screenshot 2023-08-14 at 11 38 52 AM" src="https://github.com/vercel/next.js/assets/90516185/0b150fe3-9a57-40d9-ad29-521d1eaf7a57">
b5d3e

basically when page load same API do 2 request but I not able to send 2nd request any suggestion?
